sys-apps/onerng/onerng-3.6.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit python-r1 udev
+
+DESCRIPTION="Software for the Open Hardware Random Number Generator called OneRNG"
+HOMEPAGE="https://www.onerng.info/"
+SRC_URI="https://github.com/OneRNG/onerng.github.io/raw/master/sw/${PN}_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
```

Comment:
That's not the correct license.

https://github.com/OneRNG/firmware/blob/master/cdc.c#L6

Reply:
What a mess. The Homepage itself says LGPL-3 or GPL-3, the code says LGPL-2.1 or LGPL-3. I changed it now to the later one. I didn't choose LGPL-2.1+, since it's unclear, if newer LGPL versions will be accepted by him.